awxkit/awxkit/api/pages/execution_environments.py

```diff
@@ -15,15 +15,15 @@
 log = logging.getLogger(__name__)
 
 
-class ExecutionEnvironment(HasCreate, base.Base):
+class ExecutionEnvironment(HasCreate, HasCopy, base.Base):
 
     dependencies = [Organization, Credential]
     NATURAL_KEY = ('name',)
 
-    # fields are name, image, organization, managed_by_tower, credential
-    def create(self, name='', image='quay.io/ansible/ansible-runner:devel', credential=None, **kwargs):
+    # fields are name, image, organization, managed_by_tower, credential, pull
+    def create(self, name='', image='quay.io/ansible/ansible-runner:devel', credential=None, pull='missing', **kwargs):
```

Comment:
We should almost always use `None` as a default kwarg value. See this for more info: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

I see that we were not doing that here for `image`, we should fix that too.

Reply:
Python strings aren't mutable, and are perfectly fine as default function arguments.

Reply:
pulled out into #9333